Packs/GSuiteSecurityAlertCenter/Integrations/GSuiteSecurityAlertCenter/GSuiteSecurityAlertCenter.yml

```diff
@@ -0,0 +1,805 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: G Suite Security Alert Center
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: G Suite Security Alert Center
```

Comment:
Can we supply pre-defined  types list here?

Reply:
Will check if multi-select with adding custom value is possible or not

Reply:
Added the multi-select option in alert_type parameter with support for adding custom values